assets/src/edit-story/components/form/dateTime/index.js

```diff
@@ -24,10 +37,19 @@ const DateTimeWrapper = styled.div`
 `;
 
 function DateTime({ value, onChange, is12Hour = true, forwardedRef }) {
+  const selectedTime = value ? new Date(value) : new Date();
```

Comment:
Is this ok to default value to the current date? Usually we just use an "empty" value as default. As in "no date".

Reply:
This is for opening the calendar only which will then highlight the selected date -- having the current date as the default seems to make sense in this case.

The current date should always exist though anyway -- if a story is created it will get the creation date which is then used.